src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,49 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufactureService;
+import mate.jdbc.service.ManufactureServiceImpl;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufactureService manufactureService
+                = (ManufactureService) injector.getInstance(ManufactureServiceImpl.class);
+        manufactureService.delete(1L);
+        Manufacturer peugeot = new Manufacturer();
+        peugeot.setCountry("France");
+        peugeot.setName("Peugeot");
+        manufactureService.create(peugeot);
+        Manufacturer mitsubishi = new Manufacturer();
+        mitsubishi.setName("Mitsubishi");
+        mitsubishi.setCountry("Japan");
+        mitsubishi.setId(7L);
+        manufactureService.update(mitsubishi);
+        manufactureService.delete(8L);
```

Comment:
I don't see your db, and don't see what data you have in your tables, lets imagine that your table is empty and you need to create, update and delete object inside main class, it will be more readble and uderstandable